src/components/structures/LeftPanel.js

```diff
@@ -54,6 +54,8 @@ const LeftPanel = createReactClass({
 
         this._settingWatchRef = SettingsStore.watchSetting(
             "breadcrumbs", null, this._onBreadcrumbsChanged);
+        this._settingWatchRef1 = SettingsStore.watchSetting(
```

Comment:
we should probably name these more appropriately now

something like `this._breadcrumbsWatcherRef` and `this._tagPanelWatcherRef`